api/autopilot.go

```diff
@@ -54,6 +55,7 @@ type (
 // DefaultAutopilotConfig returns a configuration with sane default values.
 func DefaultAutopilotConfig() (c AutopilotConfig) {
 	c.Wallet.DefragThreshold = 1000
+	c.Hosts.MaxDowntimeHours = 24 * 7 * 2 // 2 weeks
```

Comment:
This seems a bit low. Is this the same value we have in siad?

Reply:
I think in `skyd` it was 10 days at one point and then eventually 20 days. I think I made it two weeks because in one of our discussions I remember you saying "blabla 2 weeks", but it just seemed like a sane value to me... There's plenty of hosts I don't think we should care about hosts that are down for more than two weeks? Also they get another shot if they reannounce so..

What do you propose we set it to? I don't mind updating it.

Reply:
Ok let's keep 2 weeks for now.